dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/utils/DockerfileGitHubUtil.java

```diff
@@ -80,6 +80,15 @@ public GHRepository getRepo(String repoName) throws IOException {
         if (query.substring(query.lastIndexOf(' ') + 1).length() <= 1) {
             throw new IOException("Invalid image name.");
         }
+        // When the query string is a combination of dashes and dots, github API search doesn't work properly and doesn't find the image
+        // ex: xyz-asndnkdk-ffwwwsw-deeeereer.jfrog.io/rv-python-runtime
+        // https://github.com/craftcms/cms/issues/3080
+        // To fix this, we need to replace the dashes with spaces in the search string and Github search can then find the image
+        // New string: xyz asndnkdk ffwwwsw deeeereer.jfrog.io/rv python runtime
+        // It doesn't effect other functions and works perfectly
+        if(query.contains("-")) {
+            query = query.replace('-', ' ');
```

Comment:
I think you want to do `replaceAll`

Reply:
replaceAll fails with errors, but replace works just fine. :)

Reply:
This is what we see when I use replaceAll:
```
Error:  COMPILATION ERROR : 
Error:  /home/runner/work/dockerfile-image-update/dockerfile-image-update/dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/utils/DockerfileGitHubUtil.java:[90,38] incompatible types: char cannot be converted to java.lang.String
Error:  Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.8.1:compile (default-compile) on project dockerfile-image-update: Compilation failure
```

Reply:
**Usage:** For our case, replace should work well since we are not using regex.

String replace(char oldChar, char newChar): It replaces all the occurrences of a oldChar character with newChar character. For e.g. "pog pance".replace('p', 'd') would return dog dance.

String replaceAll(String regex, String replacement): It replaces all the substrings that fits the given regular expression with the replacement String.

Reply:
Ahh, excellent! I had a different model in my head